heron/tracker/src/python/metricstimeline.py

```diff
@@ -112,7 +110,7 @@ def getMetricsTimeline(tmaster,
 
   if metricResponse.status.status == common_pb2.NOTOK:
     if metricResponse.status.HasField("message"):
-      raise Exception(metricResponse.status.message)
+      Log.error(metricResponse.status.message)
```

Comment:
In Tornado application, if an exception is raised, it's just raised (stack trace could be printed out) but program will not terminate. If we do `Log.error`, the program continue execution and [return an empty `ret` value without metrics information](https://github.com/twitter/heron/pull/1169/files/77f9e4362294273e1bd8a2e51d6f3cea9c8f2556#diff-bb948157f88754a9d9b0b27a2df041dbR142), which is ok for the UI side.